### PR TITLE
Daniel

### PR DESCRIPTION
**Pull request by dseaton**
@antoviaque @mtyaka Added a new "current direction" example to the vector drawing tool. Also updated my repo with latest changes, so should be safe to merge new example.

### CODE REVIEW

**Comment by antoviaque** [comment]
Thanks @dseaton ! Do you want a formal review on the drawing tool, with comments/testing, or is it more FYI, to be merged directly?

**Comment by dseaton** [comment]
@antoviaque A quick look would be good - note, it is a trivial example though.

**Comment by antoviaque** [comment]
@dseaton Sounds good, I'll see who can have a look at this

File: Notes_and_Examples/5_CurrentFlow/5_currentDirection.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                  width: 13
+              },
+              vectors: [
+                    {name: 'I<sub>1</sub>', description: 'Current through R<sub>1</sub>', coords: [[4, 4], [7, 7]], render: false},
+                    {name: 'I<sub>2</sub>', description: 'Current through R<sub>2</sub>', coords: [[3, 3], [6, 6]], render: false},
+              ],
+              points: [{name: 'cm1', coords: [-0.2,-0.25]},
+                       {name: 'cm2', coords: [-0.2,-5.3]}],
+              expected_result: {
+                 'I<sub>1</sub>': {angle: 0, angle_tolerance: 5, tail: [-0.2,-0.25], tail_tolerance: 1.0},
```
Comment (e-kolpakov) [inline Notes_and_Examples/5_CurrentFlow/5_currentDirection.html]: @dseaton cool that it works, but as far as I know it's an undocumented feature - use with caution, we haven't explicitly built support for html markup in vector labels/names/etc., so warranty does not cover it :) Anyway, I believe It should just work most of the time, but if anything goes wrong - please let us know and we'll see what we can do.

File: Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<problem display_name="webGLDemo">
```
Comment (e-kolpakov) [inline Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml]: @dseaton suggestion: `display_name` isn't something magical - it's just a part of edX default markup for custom JS problem; feel free to change it to better suit your needs.

**Comment by e-kolpakov** [comment]
@dseaton I've made a couple of susggestions and sent [PR#10](https://github.com/open-craft/jsinput-vectordraw/pull/10) to slightly improve it - basically it now allows to override label on "add" button. I thought it might be useful, as "Add Selected Force" definitely does not apply to this example. 

Other than that, looks good to me. @antoviaque should I merge this?

File: Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml (hunk 2 above)
Comment (dseaton) [inline Notes_and_Examples/5_CurrentFlow/5_currentDirection.xml]: @e-kolpakov Great catch! I generally changes these in the "settings" tab in studio, but better to do when I start coding.

**Comment by dseaton** [comment]
@e-kolpakov @antoviaque Everything looks good to me. I'll merge when you merge, then update some of the files.

File: Notes_and_Examples/5_CurrentFlow/5_currentDirection.html (hunk 1 above)
Comment (dseaton) [inline Notes_and_Examples/5_CurrentFlow/5_currentDirection.html]: @e-kolpakov Don't worry, I'll be able to change this is it causes trouble. I have been using the "sub" notation extensively with JSXgraph, and it seems to work fine.

**Comment by e-kolpakov** [comment]
@dseaton I've rebased your branch on top of latest improvements - now it can be merged. You might want to double check if it works as expected - I've made a smoke test and everything seems just fine, but better be safe than sorry.

**Comment by dseaton** [comment]
@e-kolpakov I believe everything merged successfully. The only issue I found was an error in the "display name" of the current direction, but I fixed it.

**Pull request merged**